src/commonMain/resources/web/index.html

```diff
@@ -90,8 +90,56 @@ <h1 class="title mc-font">MiniMessage Viewer</h1>
+                    <span class="icon is-small"><i class="fas fa-link"></i></span>
+                  </button>
+                  <button class="button" id="editor-run-command-button" aria-label="Run Command" title="Add clickable command">
+                    <span class="icon is-small">
+                      <svg class="svg-inline--fa a-w-16" height="1em" aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+                        <path fill="currentColor" d="M339.313 2.75926L386.7 2.91021C404.492 2.91021 413.209 14.9141 404.991 30.6898L133.238 492.841L130.988 496.844C127.683 503.157 121.522 508.383 114.492 508.383L67.7287 508.443L61.7829 508.443C43.9915 508.443 35.2739 496.439 43.4916 480.663L45.4373 476.663L322.817 14.2984C326.122 7.98572 332.283 2.75926 339.313 2.75926Z"/>
+                      </svg>
+                    </span>
+                  </button>
+                  <button class="button" id="editor-suggest-command-button" aria-label="Suggest Command" title="Add clickable command suggestion">
+                    <span class="icon is-small">
+                      <svg class="svg-inline--fa a-w-16" height="1em" aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+                        <path fill="currentColor" d="M382.313 2.75926L429.7 2.91021C447.492 2.91021 456.209 14.9141 447.991 30.6898L176.238 492.841L173.988 496.844C170.683 503.157 164.522 508.383 157.492 508.383L110.729 508.443L104.783 508.443C86.9915 508.443 78.2739 496.439 86.4916 480.663L88.4373 476.663L365.817 14.2984C369.122 7.98572 375.283 2.75926 382.313 2.75926Z"/>
+                        <path fill="currentColor" d="M117.524 0C65.5586 0 31.9004 21.3336 5.47526 59.3802C0.681636 66.282 2.1595 75.7449 8.84635 80.8254L36.931 102.163C43.6842 107.294 53.293 106.094 58.5801 99.457C74.888 78.9858 86.985 67.1992 112.458 67.1992C132.487 67.1992 157.26 80.115 157.26 99.5757C157.26 114.287 145.14 121.843 125.364 132.952C102.302 145.908 71.7839 162.031 71.7839 202.365V208.75C71.7839 217.397 78.7793 224.406 87.4089 224.406H134.59C143.22 224.406 150.215 217.397 150.215 208.75V204.984C150.215 177.025 231.77 175.86 231.77 100.2C231.771 43.2217 172.785 0 117.524 0ZM111 243.624C86.1328 243.624 65.9017 263.895 65.9017 288.812C65.9017 313.728 86.1328 334 111 334C135.867 334 156.098 313.728 156.098 288.811C156.098 263.895 135.867 243.624 111 243.624Z"/>
+                      </svg>
+                    </span>
+                  </button>
+                  <button class="button" id="editor-hover-text-button" aria-label="Hover Show Text" title="Add hoverable text">
+                    <span class="icon is-small">
+                      <svg class="svg-inline--fa a-w-16" height="1em" aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```

Comment:
I don't know much, but is there any meaningful difference between inline and non-inline SVGs? If not, then this is totally fine imo.

Reply:
I was mostly concerned about ease of editing, should that be needed in the future for better icons or something

Reply:
I think images are cached for longer than HTML, so maybe it could potentially be better to extract them? But like, we're talking kilobytes here and idrc if people have internet shit enough for that to be a problem...